doc/source/cluster/cluster_under_construction/getting-started.rst

```diff
@@ -29,6 +29,13 @@ Ray clusters are officially supported on the following technology stacks:
 Advanced users may want to :ref:`deploy Ray clusters on-premise <on-prem>`
 or onto infrastructure platforms not listed here by :ref:`providing a custom node provider <ref-cluster-setup-under-construction>`.
 
+How can I run workloads on Ray clusters?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Workloads can be submitted to your cluster via :ref:`Ray Job Submission<ref-deployment-guide-under-construction>`.
+Ray clusters provide a dashboard to :ref:`monitor and observe<cluster-monitoring-and-observability>` the status of your application.
+When enabled, autoscaling is a powerful feature of Ray. See :ref:`Autoscaling<cluster-autoscaler-discussion>` for information on
+whether to enable autoscaling and how autoscaling works.
+
```

Comment:
@stephanie-wang can I get you to look at the changes in this file? They're related to your work.

Reply:
Yup, I'll modify in-place!